gtk4/src/string_object.rs

```diff
@@ -0,0 +1,97 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::StringObject;
+use glib::{GStr, GString};
+
+impl From<StringObject> for String {
+    #[inline]
+    fn from(value: StringObject) -> Self {
+        assert_initialized_main_thread!();
```

Comment:
```suggestion
        skip_assert_initialized!();
```

You can only get a `StringObject` after initialization. Also below